prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1647,6 +1647,7 @@ static size_t rxr_ep_post_ctrl(struct rxr_ep *rxr_ep, int entry_type, void *x_en
 	if (inject) {
 		err = rxr_ep_inject_pkt(rxr_ep, pkt_entry, addr);
```

Comment:
need to call rxr_handle_ctrl_sent() here

Reply:
Thanks, fixed.